providers/PaperProvider/src/main/java/net/ess3/provider/providers/PaperReflFormattedCommandAliasProvider.java

```diff
@@ -0,0 +1,22 @@
+package net.ess3.provider.providers;
+
+import net.ess3.nms.refl.ReflUtil;
+import net.ess3.nms.refl.providers.ReflFormattedCommandAliasProvider;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.FormattedCommandAlias;
+
+import java.lang.reflect.Method;
+
+public class PaperReflFormattedCommandAliasProvider extends ReflFormattedCommandAliasProvider {
+
+    @Override
+    public String buildCommand(FormattedCommandAlias command, CommandSender sender, String formatString, String[] args) {
+        try {
+            final Method buildCommandMethod = ReflUtil.getMethodCached(formattedCommandAliasClass, "buildCommand", CommandSender.class, String.class, String[].class);
```

Comment:
if you just have to use reflection anyway, I'd rather we not use the Paper provider

Reply:
Sure thing, will remove this call extra file -- however, in one of your earlier comments ( https://github.com/EssentialsX/Essentials/pull/3744#discussion_r551069304 ) you suggested I created a Paper provider for this one; would it be okay to revert this change then? or what did you have it in mind?

Reply:
Should have been more clear about this too, my point was if you need reflection for both Paper too, why do we need a Paper version too if they both do the same thing

Reply:
Ah, that's because paper appends the `CommandSender` in the buildCommand method. Are you suggesting we should just check this on the fly in the buildCommand method whether the paper / spigot version should be called?

Reply:
Okay I understand the need for that now, I think the best choice is reverting back to that one provider in that case

Reply:
Sounds good!